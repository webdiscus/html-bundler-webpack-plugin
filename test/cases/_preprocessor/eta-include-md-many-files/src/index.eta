<!DOCTYPE html>
<html>
<head>
  <title>Demo</title>
  <meta name="viewport" content="width=device-width, initial-scale=1">
  <!-- Load PrismJS styles of Highlighting from CDN, see https://cdnjs.com/libraries/prism -->
  <link href="https://cdnjs.cloudflare.com/ajax/libs/prism/9000.0.1/themes/prism.min.css" rel="stylesheet" />
  <!-- Load GitHub styles of Markdown from CDN, see https://cdnjs.com/libraries/github-markdown-css -->
  <link href="https://cdnjs.cloudflare.com/ajax/libs/github-markdown-css/5.8.1/github-markdown.min.css" rel="stylesheet" />
  <!-- Load your custom styles. -->
  <link href="./style.css" rel="stylesheet" />
</head>
<body class="markdown-body">
  <i style="color: green;">Markdown file code.md</i>
  <%~ include('/src/markdown/code.md'); %>

  <i style="color: green;">Markdown file demo.md</i>
  <%~ include('/src/markdown/demo.md'); %>
  <%~ include('/src/footer.eta', {footerText: '-= FOOTER =-'}); %>
</body>
</html>