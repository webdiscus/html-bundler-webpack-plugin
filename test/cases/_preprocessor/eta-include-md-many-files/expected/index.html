<!DOCTYPE html>
<html>
<head>
  <title>Demo</title>
  <meta name="viewport" content="width=device-width, initial-scale=1">
  <!-- Load PrismJS styles of Highlighting from CDN, see https://cdnjs.com/libraries/prism -->
  <link href="https://cdnjs.cloudflare.com/ajax/libs/prism/9000.0.1/themes/prism.min.css" rel="stylesheet" />
  <!-- Load GitHub styles of Markdown from CDN, see https://cdnjs.com/libraries/github-markdown-css -->
  <link href="https://cdnjs.cloudflare.com/ajax/libs/github-markdown-css/5.8.1/github-markdown.min.css" rel="stylesheet" />
  <!-- Load your custom styles. -->
  <link href="style.css" rel="stylesheet" />
</head>
<body class="markdown-body">
  <i style="color: green;">Markdown file code.md</i>
  <h1>Highlighting code blocks</h1>
<p>Highlight <code>Ruby</code> code:</p>
<pre class="language-"><code>```ruby
require 'redcarpet'
markdown = Redcarpet.new("Hello World!")
puts markdown.to_html
```
</code></pre>
<p>This will display the code block with syntax highlighting:</p>
<pre class="language-ruby"><code><span class="token keyword">require</span> <span class="token string-literal"><span class="token string">'redcarpet'</span></span>
markdown <span class="token operator">=</span> <span class="token class-name">Redcarpet</span><span class="token punctuation">.</span><span class="token keyword">new</span><span class="token punctuation">(</span><span class="token string-literal"><span class="token string">"Hello World!"</span></span><span class="token punctuation">)</span>
puts markdown<span class="token punctuation">.</span>to_html
</code></pre>

  <i style="color: green;">Markdown file demo.md</i>
  <h1>Markdown demo</h1>
<h1>h1 Heading</h1>
<h2>h2 Heading</h2>
<h3>h3 Heading</h3>
<h4>h4 Heading</h4>
<h5>h5 Heading</h5>
<h6>h6 Heading</h6>
<h2>Horizontal Rule</h2>
<hr>
<pre class="language-"><code>---
</code></pre>
<h2>Emphasis</h2>
<p><strong>This is bold text</strong></p>
<pre class="language-"><code>**This is bold text**
</code></pre>
<p><em>This is italic text</em></p>
<pre class="language-"><code>_This is italic text_
</code></pre>
<p><s>Strikethrough</s></p>
<pre class="language-"><code>~~Strikethrough~~
</code></pre>
<h2>Blockquotes</h2>
<blockquote>
<p>Blockquotes can also be nested...</p>
<blockquote>
<p>...by using additional greater-than signs right next to each other...</p>
<blockquote>
<p>...or with spaces between arrows.</p>
</blockquote>
</blockquote>
</blockquote>
<h2>GitHub Alerts</h2>
<p>Alerts are a Markdown extension based on the blockquote syntax.</p>
<p>Five types of alerts are available:</p>
<pre class="language-"><code>> [!NOTE]
> Useful information that users should know, even when skimming content.

> [!TIP]
> Helpful advice for doing things better or more easily.

> [!IMPORTANT]
> Key information users need to know to achieve their goal.

> [!WARNING]
> Urgent info that needs immediate user attention to avoid problems.

> [!CAUTION]
> Advises about risks or negative outcomes of certain actions.
</code></pre>
<p>Here are the rendered alerts:</p>
<blockquote class="markdown-alert markdown-alert-note"><p class="markdown-alert-title"><svg version="1.1" width="16" height="16" viewBox="0 0 16 16" class="octicon mr-2" aria-hidden=""><path d="M0 8a8 8 0 1 1 16 0A8 8 0 0 1 0 8Zm8-6.5a6.5 6.5 0 1 0 0 13 6.5 6.5 0 0 0 0-13ZM6.5 7.75A.75.75 0 0 1 7.25 7h1a.75.75 0 0 1 .75.75v2.75h.25a.75.75 0 0 1 0 1.5h-2a.75.75 0 0 1 0-1.5h.25v-2h-.25a.75.75 0 0 1-.75-.75ZM8 6a1 1 0 1 1 0-2 1 1 0 0 1 0 2Z"></path></svg> Note</p><p>
Useful information that users should know, even when skimming content.</p>
</blockquote>
<blockquote class="markdown-alert markdown-alert-tip"><p class="markdown-alert-title"><svg class="octicon octicon-light-bulb mr-2" viewBox="0 0 16 16" version="1.1" width="16" height="16" aria-hidden="true"><path d="M8 1.5c-2.363 0-4 1.69-4 3.75 0 .984.424 1.625.984 2.304l.214.253c.223.264.47.556.673.848.284.411.537.896.621 1.49a.75.75 0 0 1-1.484.211c-.04-.282-.163-.547-.37-.847a8.456 8.456 0 0 0-.542-.68c-.084-.1-.173-.205-.268-.32C3.201 7.75 2.5 6.766 2.5 5.25 2.5 2.31 4.863 0 8 0s5.5 2.31 5.5 5.25c0 1.516-.701 2.5-1.328 3.259-.095.115-.184.22-.268.319-.207.245-.383.453-.541.681-.208.3-.33.565-.37.847a.751.751 0 0 1-1.485-.212c.084-.593.337-1.078.621-1.489.203-.292.45-.584.673-.848.075-.088.147-.173.213-.253.561-.679.985-1.32.985-2.304 0-2.06-1.637-3.75-4-3.75ZM5.75 12h4.5a.75.75 0 0 1 0 1.5h-4.5a.75.75 0 0 1 0-1.5ZM6 15.25a.75.75 0 0 1 .75-.75h2.5a.75.75 0 0 1 0 1.5h-2.5a.75.75 0 0 1-.75-.75Z"></path></svg> Tip</p><p>
Helpful advice for doing things better or more easily.</p>
</blockquote>
<blockquote class="markdown-alert markdown-alert-important"><p class="markdown-alert-title"><svg class="octicon octicon-report mr-2" viewBox="0 0 16 16" version="1.1" width="16" height="16" aria-hidden="true"><path d="M0 1.75C0 .784.784 0 1.75 0h12.5C15.216 0 16 .784 16 1.75v9.5A1.75 1.75 0 0 1 14.25 13H8.06l-2.573 2.573A1.458 1.458 0 0 1 3 14.543V13H1.75A1.75 1.75 0 0 1 0 11.25Zm1.75-.25a.25.25 0 0 0-.25.25v9.5c0 .138.112.25.25.25h2a.75.75 0 0 1 .75.75v2.19l2.72-2.72a.749.749 0 0 1 .53-.22h6.5a.25.25 0 0 0 .25-.25v-9.5a.25.25 0 0 0-.25-.25Zm7 2.25v2.5a.75.75 0 0 1-1.5 0v-2.5a.75.75 0 0 1 1.5 0ZM9 9a1 1 0 1 1-2 0 1 1 0 0 1 2 0Z"></path></svg> Important</p><p>
Key information users need to know to achieve their goal.</p>
</blockquote>
<blockquote class="markdown-alert markdown-alert-warning"><p class="markdown-alert-title"><svg class="octicon octicon-alert mr-2" viewBox="0 0 16 16" version="1.1" width="16" height="16" aria-hidden="true"><path d="M6.457 1.047c.659-1.234 2.427-1.234 3.086 0l6.082 11.378A1.75 1.75 0 0 1 14.082 15H1.918a1.75 1.75 0 0 1-1.543-2.575Zm1.763.707a.25.25 0 0 0-.44 0L1.698 13.132a.25.25 0 0 0 .22.368h12.164a.25.25 0 0 0 .22-.368Zm.53 3.996v2.5a.75.75 0 0 1-1.5 0v-2.5a.75.75 0 0 1 1.5 0ZM9 11a1 1 0 1 1-2 0 1 1 0 0 1 2 0Z"></path></svg> Warning</p><p>
Urgent info that needs immediate user attention to avoid problems.</p>
</blockquote>
<blockquote class="markdown-alert markdown-alert-caution"><p class="markdown-alert-title"><svg class="octicon octicon-stop mr-2" viewBox="0 0 16 16" version="1.1" width="16" height="16" aria-hidden="true"><path d="M4.47.22A.749.749 0 0 1 5 0h6c.199 0 .389.079.53.22l4.25 4.25c.141.14.22.331.22.53v6a.749.749 0 0 1-.22.53l-4.25 4.25A.749.749 0 0 1 11 16H5a.749.749 0 0 1-.53-.22L.22 11.53A.749.749 0 0 1 0 11V5c0-.199.079-.389.22-.53Zm.84 1.28L1.5 5.31v5.38l3.81 3.81h5.38l3.81-3.81V5.31L10.69 1.5ZM8 4a.75.75 0 0 1 .75.75v3.5a.75.75 0 0 1-1.5 0v-3.5A.75.75 0 0 1 8 4Zm0 8a1 1 0 1 1 0-2 1 1 0 0 1 0 2Z"></path></svg> Caution</p><p>
Advises about risks or negative outcomes of certain actions.</p>
</blockquote>
<h2>Lists</h2>
<p>Unordered</p>
<ul>
<li>Create a list by starting a line with <code>+</code>, <code>-</code>, or <code>*</code></li>
<li>Sub-lists are made by indenting 2 spaces:
<ul>
<li>Marker character change forces new list start:
<ul>
<li>Ac tristique libero volutpat at</li>
</ul>
<ul>
<li>Facilisis in pretium nisl aliquet</li>
</ul>
<ul>
<li>Nulla volutpat aliquam velit</li>
</ul>
</li>
</ul>
</li>
<li>Very easy!</li>
</ul>
<p>Ordered</p>
<ol>
<li>Lorem ipsum dolor sit amet</li>
<li>Consectetur adipiscing elit</li>
<li>Integer molestie lorem at massa</li>
</ol>
<p>Start numbering with offset:</p>
<ol start="57">
<li>foo</li>
<li>bar</li>
</ol>
<h2>Code</h2>
<p>Inline <code>code</code></p>
<p>Indented code</p>
<pre><code>// Some comments
line 1 of code
line 2 of code
line 3 of code
</code></pre>
<p>Block code &quot;fences&quot;</p>
<pre class="language-"><code>Sample text here...
</code></pre>
<p>Syntax highlighting</p>
<pre class="language-js"><code><span class="token keyword">var</span> <span class="token function-variable function">foo</span> <span class="token operator">=</span> <span class="token keyword">function</span> <span class="token punctuation">(</span><span class="token parameter">bar</span><span class="token punctuation">)</span> <span class="token punctuation">{</span>
  <span class="token keyword">return</span> bar<span class="token operator">++</span><span class="token punctuation">;</span>
<span class="token punctuation">}</span><span class="token punctuation">;</span>

console<span class="token punctuation">.</span><span class="token function">log</span><span class="token punctuation">(</span><span class="token function">foo</span><span class="token punctuation">(</span><span class="token number">5</span><span class="token punctuation">)</span><span class="token punctuation">)</span><span class="token punctuation">;</span>
</code></pre>
<p>To display triple backticks in a fenced code block, wrap them inside quadruple backticks</p>
<pre class="language-"><code>```
Look! You can see my backticks.
```
</code></pre>
<h2>Tables</h2>
<table>
<thead>
<tr>
<th style="text-align:left">Left-aligned</th>
<th style="text-align:center">Center-aligned</th>
<th style="text-align:right">Right-aligned</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left">git status</td>
<td style="text-align:center">git status</td>
<td style="text-align:right">git status</td>
</tr>
<tr>
<td style="text-align:left">git diff</td>
<td style="text-align:center">git diff</td>
<td style="text-align:right">git diff</td>
</tr>
</tbody>
</table>
<h2>Links</h2>
<p><a href="ttps://github.com/webdiscus" title="title text!">link with title</a></p>
<pre class="language-md"><code><span class="token url">[<span class="token content">link with title</span>](<span class="token url">ttps://github.com/webdiscus</span> <span class="token string">"title text!"</span>)</span>
</code></pre>
<h2>Images</h2>
<p>External image URL</p>
<pre class="language-md"><code><span class="token url"><span class="token operator">!</span>[<span class="token content">Octocat</span>](<span class="token url">https://octodex.github.com/images/original.png</span> <span class="token string">"The Octocat original"</span>)</span>
</code></pre>
<p><img src="https://octodex.github.com/images/original.png" alt="Octocat" title="The Octocat original"></p>
<p>Local image path using Webpack alias</p>
<pre class="language-md"><code><span class="token url"><span class="token operator">!</span>[<span class="token content">Image</span>](<span class="token url">@images/octocats/plumber.webp</span> <span class="token string">"The plumber"</span>)</span>
</code></pre>
<p><img src="img/plumber.6c89dfc3.webp" alt="Image" title="The plumber"></p>
  <div>-= FOOTER =-</div></body>
</html>